storage/src/vespa/storage/storageserver/communicationmanager.cpp

```diff
@@ -102,7 +102,7 @@ CommunicationManager::handleMessage(std::unique_ptr<mbus::Message> msg)
             return;
         }
 
-        cmd->setTrace(vespalib::Trace(docMsgPtr->getTrace())); // Can it be moved ?
+        cmd->setTrace(std::move(docMsgPtr->getTrace()));
```

Comment:
Consider having a `steal_trace` function for these as well

Reply:
Done